news/_posts/2021-06-15-multistore-news-in-1.7.8.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "The multistore news in PrestaShop 1.7.8"
+subtitle: "Introducing the revamp of the multistore”
+date:   2021-06-15 16:00:00
+authors: [MarionFrancois]
+icon: icon-lightning
+tags:
+ - 1.7.8
+ - version
+ - minor
+ - product
+ - ux
+---
+
+With the release of the new version 1.7.8 which is in progress, this article details one of the main features revamps of this version: the multistore.
+
+The multistore allows merchants to manage several front offices through a single back office. It’s frequently used by merchants who sell in different countries and therefore have different languages and/or currencies. It is also used by merchants who do both B2B and B2C. And it is more rarely used by merchants who have several brands.
+
+We are thrilled to introduce to you the new features and changes around the multistore. But before jumping into the features awaiting our users in this new version, we would like to share our research and conception process.
+
+**Firstly, why rethinking this feature?**
+
+
+The multistore was introduced in PrestaShop 1.5 and had not been reworked in many years. 
+Also besides, some features of the multistore having disappeared with the Symfony migration. This feature was becoming particularly unattractive and hard to use in the back-office also because of its old interface that led to misunderstanding and configuration errors for our users.
```

Comment:
```suggestion
In addition, some features of the multistore having disappeared with the Symfony migration, this feature was becoming particularly unattractive. It was also hard to use in the back office because of its old interface that led to misunderstanding and configuration errors for users.
```